tests/algorithms/test_preference_comparisons.py

```diff
@@ -202,22 +202,22 @@ def test_trainer_no_crash(
     assert 0.0 < result["reward_accuracy"] <= 1.0
 
 
-def test_reward_ensemble_trainer_raises_type_error(venv):
```

Comment:
I think the `RewardEnsemble` is still raising the type error in this implementation.